### PR TITLE
[MM-44438]: fix Advanced Text editor Padding issue in RHS and Thread view

### PR DESCRIPTION
**Pull request by AshishDhama**
#### Summary
fix Advanced Text editor Padding issue in RHS and Thread view

#### Ticket Link
  Fixes https://mattermost.atlassian.net/browse/MM-44438
  Fixes https://mattermost.atlassian.net/browse/MM-44609

#### Related Pull Requests
NONE

#### Screenshots
Screenshot 2022-06-03 at 5 04 24 PM

#### Release Note
```release-note
fix Advanced Text editor Padding issue in RHS and Thread view
```

### CODE REVIEW

**Review by vish9812 — APPROVED** [review]

**Comment by amyblais** [comment]
@jfrerich Please help prioritize this PR review.

File: components/advanced_text_editor/advanced_text_editor.scss [hunk 1]
```diff
@@ -171,6 +179,18 @@
                 border-left: none;
             }
         }
+
+        .ThreadViewer & {
+            .msg-typing:empty {
```
Comment (michelengelen) [inline components/advanced_text_editor/advanced_text_editor.scss:184]: this is fine, but keep in mind that this selector is super flaky regarding white-spaces and inner elements, even if they are not showing/displaying or have no content on their own

**Review by michelengelen — APPROVED** [review]
LGTM

File: components/advanced_text_editor/advanced_text_editor.scss (hunk 1 above)
Comment (AshishDhama) [reply components/advanced_text_editor/advanced_text_editor.scss:184]: @michelengelen are you referring to empty pseudo selector.

Comment (michelengelen) [reply components/advanced_text_editor/advanced_text_editor.scss:184]: yes, I am referring to that ... I had problems with that in the past, but when we are careful with what we insert in that `.msg-typing` element we are good to go!

**Comment by jgilliam17** [comment]
Thanks @AshishDhama 
Tested, looks good to merge
- Verified padding between last message in the thread and the textbox - as expected in both standard and compact message view.
- [E2E report ](https://mattermost-cypress-report.s3.amazonaws.com/203773-b939269-onprem-ent-webapp-pr-10466-mattermost/mm-ee-testb939269/mochawesome.html)- no PR related failures.

**Review by jgilliam17 — APPROVED** [review]

**Pull request merged**